src/StoryDeck.elm

```diff
@@ -57,21 +64,14 @@ storyTeaser deckId =
     let
         t =
             toString
+
+        works =
+            { host = "www.google.com", protocol = Http }
+                |> Url.Url
     in
-    div [ class "card" ]
-        [ a
-            [ href ("#/stories/" ++ String.fromInt deckId)
-            , onClick (ButtonPress "story" "view-single" (t (storyTitle deckId)) False)
-            ]
-            [ img [ class "card--thumbnail", src (storyTeaserImgPath deckId), alt (t (storyTeaserImgAltText deckId)) ] []
-            ]
-        , a
-            [ class "link--unstyled"
-            , href ("#/stories/" ++ String.fromInt deckId)
-            , onClick (ButtonPress "story" "view-single" (t (storyTitle deckId)) False)
-            ]
-            [ h3 [ class "title--small" ] [ text (t (storyTitle deckId)) ]
-            ]
+    div [ class "card", onClick (UrlChanged works) ]
```

Comment:
If we do want onClick here then using the same ButtonPress message as before should be ok - rather than the UrlChange stuff.

Also - this gives us the issue of having the whol thing clickable which makes it potentially more difficult for the majority of our users on touch screens.